spec/factories/kudos.rb

```diff
@@ -4,5 +4,6 @@
     content { 'this is my first kudo created by a factory' }
     giver factory: :employee
     receiver factory: :employee
+    company_value_id { Faker::Adjective.positive }
```

Comment:
this factory will try to create Kudo with some kind of `String` in column `company_value_id` that accepts integers, therefore it will break.